src/antares/service/api_services/renewable_api.py

```diff
@@ -63,3 +64,23 @@ def get_renewable_matrix(self, renewable: RenewableCluster) -> pd.DataFrame:
             return get_matrix(f"{self._base_url}/studies/{self.study_id}/raw?path={path}", self._wrapper)
         except APIError as e:
             raise RenewableMatrixDownloadError(renewable.area_id, renewable.name, e.message) from e
+
+    def read_renewables(
+        self,
+        area_id: str,
+    ) -> List[RenewableCluster]:
+        json_renewables = self._wrapper.get(
+            self._base_url + "/studies/" + self.study_id + "/areas/" + area_id + "/clusters/renewable"
```

Comment:
Minor change for the 3 services, you could use a f-string as it's easier to read:
`url = f"{self._base_url}/studies/{self.study_id}/areas/{area_id}/clusters/renewable"`
`json_renewables = self._wrapper.get(url).json()`